### PR TITLE
Cleanup and other non-fixes for two (and three)

### PR DESCRIPTION
**Pull request by albertvaka**
Best reviewed commit by commit, since one of the commits moves code around.

### CODE REVIEW

File: six/two/two.cpp [hunk 1]
```diff
@@ -830,7 +838,6 @@ void Two::setIsExcludedCb(cb_is_excluded_t cb)
 // Python Helpers
 
 // get_integration_list return a list of every datadog's wheels installed. The
```
Comment (KSerrania) [inline six/two/two.cpp]: You forgot to remove the "The" at the end of the line which belonged to the sentence you removed.

File: six/three/three.cpp [hunk 2]
```diff
@@ -89,7 +89,7 @@ bool Three::init()
     Py_Initialize();
 
     // Set PYTHONPATH
-    if (_pythonPaths.size()) {
+    if (!_pythonPaths.empty()) {
```
Comment (truthbk) [inline six/three/three.cpp]: 🍰

File: six/common/builtins/aggregator.c [hunk 3]
```diff
@@ -95,7 +94,7 @@ static char **py_tag_to_c(PyObject *py_tags)
         return tags;
     }
 
-    py_tags_list = PySequence_Fast(py_tags, err); // new reference
+    py_tags_list = PySequence_Fast(py_tags, "py_tags is not a sequence"); // new reference
```
Comment (truthbk) [inline six/common/builtins/aggregator.c]: We should maybe start adding/consolidating error message strings, into header(s) for consistency (as a best practice). It's a bit of a chore and maybe beyond the scope of this PR, we could tackle that separately.

**Review by truthbk — DISMISSED** [review]
This looks good to me, thank you for all this cleanup.

**Pull request merged**